strawberry/object_type.py

```diff
@@ -193,6 +200,7 @@ def type(
     is_interface: bool = False,
     description: Optional[str] = None,
     directives: Optional[Sequence[object]] = (),
+    extensions: Optional[List[TypeExtension]] = None,
```

Comment:
lame nitpick -> should at least follow directies and have a default of empty sequence. e.g.

```
directives: Optional[Sequence[object]] = (),
extensions: Optional[Sequence[TypeExtension]] = (),
```
though idk why we even needed the None case, it could just be 
```
extensions: Sequence[TypeExtension]] = (),
```